### ISSUE TITLE
Seams visible on lightmapped cylinder

### ISSUE THREAD

**Issue by willeastcott** [playcanvas/engine#4142]
![image](https://user-images.githubusercontent.com/697563/159114751-96463c92-765f-4603-8522-565ad4da198c.png)

This seems to occur when baking ambient from skybox. Disabling ambient baking gives:

![image](https://user-images.githubusercontent.com/697563/159114804-13338fab-96f9-47b7-8a94-917bd6d51ef6.png)

Scene: https://playcanvas.com/editor/scene/1366704

**Comment by mvaligursky** [comment]
Yep, we're aware of these and discussed them before - this is a typical problem with lightmapping seams across the uv boundaries. There are CPU based solutions to blur across the seams, but nothing easy to do on GPU that I could find. This is even worse when lightmap blur filter is enabled, as it does not blur across the seams.
